packages/create-amplify/templates/basic-auth-data/amplify/data/resource.ts

```diff
@@ -1,12 +1,58 @@
 import { type ClientSchema, a, defineData } from '@aws-amplify/backend';
 
+/*== STEP 1 ===============================================================
+The section below creates a Todo database table with a "content" field. Try
+adding a new "isDone" field as a boolean. The authorization rules below
+specify that owners, authenticated via your Auth resource can "create",
+"read", "update", and "delete" their own records. Public users,
+authenticated via an API key, can only "read" records.
+=========================================================================*/
 const schema = a.schema({
-  Todo: a.model({
-    name: a.string(),
-    description: a.string(),
-  }),
+  Todo: a
+    .model({
+      content: a.string(),
+    })
+    .authorization([a.allow.owner(), a.allow.public().to(['read'])]),
 });
 
 export type Schema = ClientSchema<typeof schema>;
 
-export const data = defineData({ schema });
+export const data = defineData({
+  schema,
+  authorizationModes: {
+    defaultAuthorizationMode: 'API_KEY',
+    // API Key is used for a.allow.public() rules
```

Comment:
:-)

Reply:
We should have opened this PR a day earlier 😄